.github/workflows/c-cpp.yml

```diff
@@ -96,6 +111,15 @@ jobs:
           sudo make -C SDL2-${SDL2VER} install
         if: matrix.os == 'ubuntu-latest'
 
+      - name: Set up linuxdeploy (ubuntu-latest)
+        run: |
+          if [[ ! -e linuxdeploy-x86_64.AppImage ]]; then
+            curl -sLO https://github.com/linuxdeploy/linuxdeploy/releases/latest/download/linuxdeploy-x86_64.AppImage
+          fi
+          sudo cp -f linuxdeploy-x86_64.AppImage /usr/local/bin/
+          sudo chmod +x /usr/local/bin/linuxdeploy-x86_64.AppImage
+        if: matrix.os == 'ubuntu-latest'
```

Comment:
Isn't this only needed for the appimage build?

Reply:
Fixed.